lib/Core/Memory.h

```diff
@@ -461,8 +454,8 @@ class PersistentState : public ObjectState {
     static ref<ConstantExpr> getNullptr();
 
   private:
-    ref<Expr> isCacheLinePersisted(unsigned offset) const;
-    ref<Expr> isCacheLinePersisted(ref<Expr> offset) const;
+    ref<Expr> isCacheLinePersisted(unsigned offset, bool pending=false) const;
```

Comment:
Is the "pending" for the bug reporting stuff I mentioned above? If so, can you add a comment?

Reply:
`pending` here has the same meaning as in `getIsOffsetPersistedExpr`, which is commented as follows:

```
* Get an expression which will evaluate to 1 if all writes to the given
* offset have been persisted and fenced.
* If pending=true, then expression will also evaluate to 1 even if not fenced
* (used to detect redundant flushes).
```